projects/fsastorefrontlib/src/cms-components/product/product.module.ts

```diff
@@ -12,7 +12,6 @@ import { RouterModule } from '@angular/router';
 import {
   MediaModule,
   ListNavigationModule,
-  FacetListModule,
   ProductFacetNavigationModule,
```

Comment:
This `ProductFacetNavigationModule` is also related to facets. You could remove it since we don't display facets in the product-list component.